src/HelloServlet.java

```diff
@@ -0,0 +1,64 @@
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.*;
+
+@WebServlet("/HelloServlet")
+
+
+public class HelloServlet extends HttpServlet {
+
+
+    private String globalVar = null;
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
```

Comment:
Why this method `protected` ? This class do not extends by other classes, so do it `public`.